src/test/rustdoc-ui/issue-79494.stderr

```diff
@@ -0,0 +1,12 @@
+error[E0512]: cannot transmute between types of different sizes, or dependently-sized types
+  --> $DIR/issue-79494.rs:9:29
+   |
+LL | const ZST: &[u8] = unsafe { std::mem::transmute(1usize) };
+   |                             ^^^^^^^^^^^^^^^^^^^
+   |
+   = note: source type: `usize` (64 bits)
+   = note: target type: `&[u8]` (128 bits)
```

Comment:
Presumably this will need similar normalization to work across platforms, though an `// only-x86_64-unknown-linux-gnu` might be better in some ways.